src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/LambdaCompiler.Expressions.cs

```diff
@@ -862,24 +860,6 @@ private void EmitMemberGet(MemberInfo member, Type objectType)
             }
         }
 
-        private static bool TryGetRawConstantValue(FieldInfo fi, out object value)
-        {
-            // TODO: It looks like GetRawConstantValue is not available at the moment, use it when it comes back.
-            //value = fi.GetRawConstantValue();
-            //return true;
-
-            try
-            {
-                value = fi.GetValue(obj: null);
-                return true;
-            }
-            catch
-            {
-                value = null;
```

Comment:
I do not think we need the exception handling. There is really no way to handle a failure in GetRawConstantValue.
Once a field claims to be a literal, you must emit its constant value.